subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ReportState.kt

```diff
@@ -3,112 +3,151 @@ package com.avito.android.test.report
 import com.avito.android.test.report.model.DataSet
 import com.avito.android.test.report.model.StepResult
 import com.avito.android.test.report.model.TestMetadata
+import com.avito.filestorage.FutureValue
+import com.avito.filestorage.RemoteStorage
 import com.avito.report.model.Entry
 import com.avito.report.model.Incident
 import com.avito.report.model.Video
 
 sealed class ReportState {
 
-    abstract val isFirstStepOrPrecondition: Boolean
```

Comment:
Delete because only one state could answer that question
Look impl in ReportImplementation